finagle-core/src/main/scala/com/twitter/finagle/ssl/TrustCredentials.scala

```diff
@@ -34,4 +35,12 @@ object TrustCredentials {
    * in PEM format.
    */
   case class CertCollection(file: File) extends TrustCredentials
+
+  /**
+    * The collection of certificates which should be used in
+    * verifying a remote peer's credentials.
+    *
+    * @certs a collection of X.509 certificates
+    */
+  case class Certificates(certs: Array[X509Certificate]) extends TrustCredentials
```

Comment:
Undo this change please. If you would like to add this outside the context of this pull request, we can discuss it in a separate PR.